concepticondata/concepticon.tsv

```diff
@@ -3751,3 +3751,4 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3765	STORING POT	Food and drink	A vessel used for storing food or liquids.	Person/Thing	
 3766	WATER POT	Food and drink	A vessel used for storing water.	Person/Thing	
 3767	RICE SEEDLING	Agriculture and vegetation	The seedling of a rice plant.	Person/Thing	
+3768	POUND (FOOD)	Motion	The preparation of food with a pounding or beating moving.	Action/Process	
```

Comment:
I would change that to "To prepare food...".

Reply:
Oh, and "movement" instead of "moving". And as implied below, I would specify that it's about the preprocessing of some grain or of sago.